### DISCUSSION TITLE
👋 Welcome to daily.dev Discussions!

### DISCUSSION THREAD

**Discussion by nimrodkra** [dailydotdev/daily#224] [General]
## 🦄 Welcome to daily.dev GitHub Discussions!
  We’re using Discussions to create meaningful conversations around daily.dev. It is a place for our community to connect with each other around ideas, questions, issues, and suggestions. We hope that you:
  * Ask questions you’re wondering about.
  * Share your ideas on how we can improve.
  * Engage with other community members.
  * Respectful towards other members and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 🎸 Feature requests
We love building stuff. But it's much more meaningful when it's something people want! Feel free to send us ideas and perhaps even contribute to the daily.dev project. Here's the [ideas channel](https://github.com/dailydotdev/daily/discussions?discussions_q=category%3AIdeas) where you can share new feature requests. 

## 🔗 Submit an issue
Found something that bothered you and you'd like to submit an issue? That's a great place to start. Here's a link to the [issue template](https://github.com/dailydotdev/daily/blob/master/.github/ISSUE_TEMPLATE/---bug-report.md).

## 🤗 Make sure you never miss important updates about daily.dev
* Click on the ⭐️ and 👁 button above ;)
* Follow us on Twitter at [@dailydotdev](https://twitter.com/dailydotdev)
* Check out [our blog](https://daily.dev/blog)

## 🌈 Did you miss anything?
Shoot us an email at hi@daily.dev and a real human from our core team will reply to you as soon as possible.

**Comment by Minecodes** [comment]
I think, that was an nice idea 😉

**Reply by Berkmann18** [reply]
Agreed.

**Comment by Isaac-Tait** [comment]
Ooh exciting! I am intrigued to see where this will take the daily.dev community

**Comment by Berkmann18** [comment]
😁
